monky.el

```diff
@@ -2371,8 +2371,6 @@ With a non numeric prefix ARG, show all entries"
   :lighter ()
   :keymap monky-blame-mode-map)
 
-(defvar monky-blame-buffer-name "*monky-blame*")
```

Comment:
I'm wondering if we should let this being configurable. Do you think some people change this?

Reply:
I can't think of anyone wanting to change this, nor of equivalent packages that let you fiddle with buffer names. Users could always advise `pop-to-buffer` if they really wanted to change the name.

This isn't a `defcustom` so I suspect it wasn't originally intended for being configurable.